src/logic/operate.js

```diff
@@ -0,0 +1,29 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const x = Big(numberOne);
+  const y = Big(numberTwo);
+  let r = 0;
+  switch (operation) {
+    case '+':
+      r = x.plus(y);
+      break;
+    case '-':
+      r = x.minus(y);
+      break;
+    case 'x':
+      r = x.times(y);
+      break;
+    case '÷':
+      r = x.div(y);
+      break;
+    case '%':
+      r = x.div(100);
```

Comment:
- [ ] `Operate` switches `operation` and uses `plus`, `minus`, `times`, `div` and `modifier` (turning the number into a percentage based number, `e.g. 95 % == .95`) functions ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone4/performOperate.png)) (It is required that you specifically use the .mod method from the big.js package, please do so to comply with the project requirements)